packages/@aws-cdk/aws-ec2/lib/machine-image.ts

```diff
@@ -468,7 +499,7 @@ export enum AmazonLinuxStorage {
   /**
    * S3-backed storage
    */
-  S3 = 'ebs',
+  S3 = 's3',
```

Comment:
@njlynch Was this an omission... or by design?

Reply:
I imagine this was a mistake. Somehow it's gone unnoticed for 3 years!

Reply:
😅